fares-data-build-tool/fdbt-site
# Feature/tfn 406 flat fare user journey

# Description

-   This PR introduces a new user journey for entering a Flat Fare Product.
-   This PR also includes some housekeeping to improve consistency between web pages

# Testing instructions

-   Pull down this branch and run all resources locally using the make command in fdbt-dev
-   Run through the new Flat Fare Product user journey (see the sitemap in https://drive.google.com/drive/u/0/folders/0AFrrMIylp35DUk9PVA for more detail) and validate that the 'fareType', 'serviceList' and 'productDetails' pages route correctly.
-   Run through the existing Period Ticket user journeys to validate that the existing pages route as they did in the past.
- Run `awslocal s3api list-objects --bucket fdbt-matching-data-dev` to find the file which has been dumped in s3local by the different user journeys. Then run `awslocal s3api get-object --bucket fdbt-matching-data-dev --key <FILENAME IN S3LOCAL> <FILEPATH TO SAVE TO>`

# Type of change


# Checklist:


<img width="1637" alt="Screenshot 2020-05-15 at 15 21 06" src="https://user-images.githubusercontent.com/55277550/82061056-1f39f280-96c0-11ea-82d7-7a8e4362859a.png">
<img width="1646" alt="Screenshot 2020-05-15 at 15 21 41" src="https://user-images.githubusercontent.com/55277550/82061064-219c4c80-96c0-11ea-8408-374a7b9ba794.png">
<img width="1636" alt="Screenshot 2020-05-15 at 15 23 27" src="https://user-images.githubusercontent.com/55277550/82061067-2234e300-96c0-11ea-844c-9f047f126f92.png">
<img width="1638" alt="Screenshot 2020-05-15 at 15 23 51" src="https://user-images.githubusercontent.com/55277550/82061068-22cd7980-96c0-11ea-8104-a39d7c125c6c.png">
